audit/projects/kubernetes-public/services/container/clusters/aaa.json

```diff
@@ -102,7 +102,7 @@
   "nodeConfig": {
     "diskSizeGb": 100,
     "diskType": "pd-standard",
-    "imageType": "COS",
+    "imageType": "COS_CONTAINERD",
```

Comment:
Expected, deploy of https://github.com/kubernetes/k8s.io/pull/2252

(same for the node pools below)